docs/pages/visual-test/RadioGroup-light.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import { type Node } from 'react';
+import { RadioGroup, ColorSchemeProvider } from 'gestalt';
+
+export default function Snapshot(): Node {
+  return (
+    <ColorSchemeProvider colorScheme="light">
```

Comment:
This snap should be updated. The image is incorrect. Also can we add 

`      <Box color="white" display="inlineBlock" padding={1}>
`

wrapping the component so the image has some padding?

I'd would strongly suggest using a simple example rather than testing as this images are displayed in VSCOde to all engineers